frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -131,48 +115,75 @@ const ObjectDashboardBucketsCard: React.FC<DashboardItemProps> = ({
   const obcLoaded = _.get(resources.obc, 'loaded');
   const obcLoadError = _.get(resources.obc, 'loadError');
 
-  const noobaaStorageClassList = getNoobaaStorageClassList(storageClassesData);
-  const noobaaObcList = getNoobaaObcList(obcData, noobaaStorageClassList);
+  const obData = _.get(resources.ob, 'data', []) as K8sResourceKind[];
+  const obLoaded = _.get(resources.ob, 'loaded');
+  const obLoadError = _.get(resources.ob, 'loadError');
 
-  const obcProvisioningFailure = noobaaObcList.filter((obc) => !isBound(obc)).length;
-
-  const bucketProps: BucketsItemProps = {
-    title: 'Noobaa Bucket',
-    bucketsCount: +getGaugeValue(obCountResponse),
-    objectsCount: getGaugeValue(obObjectsCountResponse),
-    unhealthyCounts: [+getGaugeValue(unhealthyObResponse)],
-    isLoading: !(obCountResponse && obObjectsCountResponse && unhealthyObResponse),
-    hasLoadError: obCountResponseError || obObjectsCountResponseError || unhealthyObResponseError,
-    links: [noobaaBucketsLink],
-  };
+  const noobaaStorageClassList = getNoobaaStorageClassList(storageClassesData);
+  const obcObjects = getGaugeValue(obcObjectsCount);
+  const noobaaObjects = getGaugeValue(noobaaObjectsCount);
 
-  const bucketClaimProps: BucketsItemProps = {
-    title: 'Object Bucket Claim',
-    bucketsCount: noobaaObcList.length,
-    objectsCount: getGaugeValue(obcObjectsCountsResponse),
-    unhealthyCounts: [+getGaugeValue(unhealthyObcCountResponse), obcProvisioningFailure],
-    isLoading: !(
-      storageClassesLoaded &&
-      obcLoaded &&
-      obcObjectsCountsResponse &&
-      unhealthyObcCountResponse
-    ),
-    hasLoadError:
-      storageClassesLoadError ||
-      obcLoadError ||
-      obcObjectsCountsResponseError ||
-      unhealthyObcCountResponseError,
-    links: [noobaaBucketsLink, resourcePathFromModel(NooBaaObjectBucketClaimModel)],
-  };
+  const obcLink = resourcePathFromModel(NooBaaObjectBucketClaimModel);
+  const obLink = resourcePathFromModel(NooBaaObjectBucketModel);
 
   return (
     <DashboardCard>
       <DashboardCardHeader>
         <DashboardCardTitle>Buckets</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
-        <BucketsItem {...bucketProps} />
-        <BucketsItem {...bucketClaimProps} />
+        <InventoryItem
+          isLoading={!(storageClassesLoaded && noobaaCount)}
```

Comment:
Noobaa count can be 0 when Noobaa is still installing. This will lead to loading showing up.

Reply:
`noobaaCount` is a promethues response, so it wont be 0.